amazon-kinesis/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,73 @@
+= Amazon Kinesis Data Streams Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Amazon Kinesis Data Streams (Amazon Kinesis Data Streams Connector)
+
+This connector provides a single common configuration that enables access to reading and writing operations for an Amazon Kinesis data stream.
+
+For compatibility information and fixed issues, see the Amazon Kinesis Data Streams Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Amazon Kinesis Data Streams API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* AWS Identity and Access Management (IAM) credentials
```

Comment:
This should be rephrased to `AWS IAM Access Key credentials` and maybe with added ` with access rights to Kinesis, DynamoDb and CloudWatch services.`